src/composables/useApp.ts

```diff
@@ -73,13 +66,12 @@ export function useApp() {
     return;
   }
 
-  const { domain } = useDomain();
   const { apolloQuery } = useApolloQuery();
 
   const skin = ref('');
 
   async function getSkin() {
-    const key = aliases[domain] || domain;
+    const key = domain;
```

Comment:
This change is because skin name must always be original domain and not the alias?

Reply:
Yes, not sure why this was here in the first place